workflow/controller/operator_test.go

```diff
@@ -3394,6 +3394,31 @@ spec:
 	}
 }
 
+// createRunningPods creates the pods that are marked as running in a given test so that they can be accessed by the
+// pod assessor
+func createRunningPods(ctx context.Context, woc *wfOperationCtx) {
```

Comment:
Can you move it to controller_test so we refer  common function in one place/

Reply:
Sure. Moved it back to original location.